### PR TITLE
Add project badges (fixes #340)

### LINKED ISSUE #340: Active || Maintained || Archived - projects badges
#### Subject of the issue:
We could add badges to our projects list so that it's clear which of them a person can actively join.

### PR DESCRIPTION
**Pull request by OtisRed**
#### Story / Bug id:
#340 

#### Description:
I added temporary solution for project badges. Our projects component will be redesigned anyway, but until UX team will provide mock up it may serve as a working solution. 

#### New imports / dependencies:
N/A

#### What tests do I need to run to validate this change:
Visual ones

### CODE REVIEW

**Comment by KSodowska** [comment]
@OtisRed one suggestion - maybe instead of putting this status information on the miniature, we should include this in the pop-up window after you click on it, where other informations are visible? Just a thought.

File: frontend/src/components/OurProjects.vue [hunk 1]
```diff
@@ -81,6 +88,33 @@ export default {
...
+  left: 0.75rem;
+  top: 0.75rem;
+  height: 2rem;
+  padding: 0.25rem 0.75rem;
+  border-radius: 25px;
+  box-shadow: 1px 3px #2c3e50;
+  font-family: $font-content;
+  font-size: 1.2rem;
+  text-transform: capitalize;
+  font-weight: bold;
```
[the remark this answers is not in the document]
Comment (OtisRed) [reply frontend/src/components/OurProjects.vue]: Acknowledged!

File: frontend/src/components/OurProjects.vue [hunk 2]
```diff
@@ -81,6 +88,33 @@ export default {
...
+}
+
+.card--badge_status--aktywny {
```
[the remark this answers is not in the document]
Comment (OtisRed) [reply frontend/src/components/OurProjects.vue]: I think it's more elegant now :)

File: frontend/src/components/OurProjects.vue [hunk 3]
```diff
@@ -23,7 +23,14 @@
         <v-item-group>
           <v-hover v-slot="{ hover }">
             <v-card @click.stop="clickImage(project)">
-              <v-img :src="project.imageAdress" aspect-ratio="1.9" />
+              <v-img :src="project.imageAdress" aspect-ratio="1.9">
+                <div
+                  class="card--badge"
+                  :class="'card--badge_status--' + project.badge"
```
[the remark this answers is not in the document]
Comment (OtisRed) [reply frontend/src/components/OurProjects.vue]: Done!

**Comment by OtisRed** [comment]
@Marce1ina @arturtamborski - do you have any ideas how to solve [this problem](https://github.com/CodeForPoznan/codeforpoznan.pl_v3/actions/runs/3145841440/jobs/5113611376)? 
1. I tried installing lodash but without success... 
1. Also it's big so I tried to install only lodash.orderby but I wasn't able to make it work without this damn underscore. 
1. At the same time I can't think of how to sort this better while generating array through v-for (it was the [recommended solution](https://v2.vuejs.org/v2/guide/migration.html?redirect=true#Replacing-the-orderBy-Filter))

Any suggestions?

**Review by w1stler — APPROVED** [review]

**Pull request merged**